os/mm/mm_heap/mm_getheap.c

```diff
@@ -219,6 +219,11 @@ int mm_get_heapindex(void *mem)
 	if (mem == NULL) {
 		return 0;
 	}
+	if(BASE_HEAP == NULL) {
```

Comment:
1. Please add a space after if
2. Is this possible case? If BASE_HEAP is NULL, is it possible to come here?

Reply:
1. ok
2. It will help during porting and easy debug, we have faced similar issue during current loadable porting